doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,262 @@
+
+```
+Press enter to set-up an Account A with funds.
+New market account after deposit:
+  * tETH: 0 -> 10.0
+  * tUSDC: 0 -> 10000.0
+Account A was set-up: 0xb5a2e0363ecfd9fbc9cd9e77f5db5d7d6e85a1c8b931675f71e99a1836adfe41
+TRANSACTIONS EXECUTED (first-to-last):
+  * Mint 10.0 tETH (yet to be deposited): 0xbecafd3453798384eacdc25733d53e30d8d86524c27cf2f8da533a3ad785838a
+  * Mint 10000.0 tUSDC (yet to be deposited): 0xfdddda8daf3881908c413ec5d8cf12fb3a2d164d57a43908b06236171cfd1103
+  * Register a new account in market 1: 0x89a403548b2b986555751fa0e4d2d29e6a099ca38dd2d5640297b587fb27510c
+  * Deposit 10.0 tETH to market account: 0x8233bc577deafd9f3ace59c9d7843807c1a41bf9604e980eddff47990d677bdc
+  * Deposit 10000.0 tETH to market account: 0xd6b6eeb06160c13c174ea4c868fef9457686f45c7839463640cb7a5b051f95a3
+```
+
+Interacting with the exchange as an address requires a market account for each trading pair. Market accounts possess the funds available to trade on a trading pair as an account. In the above, we perform two steps for two coins, in addition to registering a new market account:
+
+1. Create a new market account (only if necessary).
+2. Obtain the funds to be deposited (in this case, we mint them).
+3. Depost the funds into the account.
+
+This gives us the 5 total transactions we expect and see above.
+
+#### Step #3: Place Limit Orders (as Account A)
+```
+Press enter to place limit orders with Account A.
+EVENT SUMMARY: PlaceLimitOrderEvent
+  * User address: 0xb5a2e0363ecfd9fbc9cd9e77f5db5d7d6e85a1c8b931675f71e99a1836adfe41
+  * Order ID: 18446884819787842536
+  * Side: BID (Buying)
+  * Price: 1000 tUSDC ticks per tETH lot
+  * Size: 1000 available tETH lots / 1000
+EVENT SUMMARY: PlaceLimitOrderEvent
+  * User address: 0xb5a2e0363ecfd9fbc9cd9e77f5db5d7d6e85a1c8b931675f71e99a1836adfe41
+  * Order ID: 36893628897792362448
+  * Side: ASK (Selling)
+  * Price: 2000 tUSDC ticks per tETH lot
+  * Size: 1000 available tETH lots / 1000
+Account A has finished placing limit orders.
+  * There were no limit orders filled by any orders placed.
+TRANSACTIONS EXECUTED (first-to-last):
+  * Place limit BID/BUY order (1000 lots) (1000 ticks/lot): 0x7f8a086a31a7298f68eb474179de845ca31f0c124a31bdf3ecbdad23d571d880
+  * Place limit ASK/SELL order (1000 lots) (2000 ticks/lot): 0xc994c5aca37a83aca9514b002d54788650943cba80d008c48ffa2870898fbf12
+CURRENT BEST PRICE LEVELS:
+  * Highest BID/BUY @ 1000 ticks/lot, 1000 lots
+  * Lowest ASK/SELL @ 2000 ticks/lot, 1000 lots
+```
+
+Limit orders behave as expected: some of the user's market account funds are "locked up" in what's called a limit order, as opposed to a market order. Limit orders make an asset available for purchase at a given price. When a market order comes along and pays the price of a limit order, this is called a "fill" event. We're about to witness such a fill event in the coming steps.
+
+#### Step #4: Set-up the Account "B"
+
+```
+Press enter to set-up and Account B with funds.
```

Comment:
It looks like there might be a minor typo in the trade script: "and" vs "an"

By contrast the one for Account A says "Press enter to set-up an Account A with funds."

So I think you need to modify the trade script print line and then just take out the "d" here